cmake/Modules/FindSYCL/run_sycl.cmake

```diff
@@ -108,10 +113,16 @@ SYCL_execute_process(
   )
 
 # Generate the code
+set(SYCL_DEPENDENCY_FLAGS "")
+# TODO: enable incremental build on Windows
+if(NOT WIN32)
```

Comment:
`NOT MSVC`? since `MD` is a feature of MSVC compiler?

Reply:
Thanks, updated.